data/techs_str.dat

```diff
@@ -132,22 +132,22 @@ result = I can now construct computing Clusters tailored for my use.
 [Personal Identification]
 name = Personal Identification
 description = This world requires identification for most tasks.  Fortunately, most databases are woefully insecure.  I should be able to construct identifying material in enough databases to pass routine checks on my (faked) existence as a human being.
-result = With my newfound identities, I can now work Basic Jobs, rent Storage Units and space in Datacenters.
+result = With my newfound identities, I can now work Basic Jobs and rent space in both Storage Units and Datacenters.
 
 [Advanced Personal Identification]
 name = Advanced Personal Identification
 description = Further manipulation of personnel databases should make my existence more convincing.  Some of these databases are heavily protected; those will require time and persistence to compromise, but the result should be worth it.
-result = I am now convincingly human enough to contract the construction of Small Warehouses.
+result = I am now convincingly human enough to contract for the construction of Small Warehouses.
```

Comment:
I do not think I have ever seen the phrase `to contract for`. When I look it up at https://idioms.thefreedictionary.com/contract+for , then it suggests `contract with`

Reply:
Yeah, I'm tweaking that.